### ISSUE TITLE
Warn if "SC" font has .sc or .smcp suffixes in glyphs

### ISSUE THREAD

**Issue by thundernixon** [fonttools/fontspector#451]
Probably not a high-priority check; just a font issue I came across this week.

## Observed behavior

Playfair Display SC correctly sets small cap glyphs as the default `/a /b /c` etc, but then also has the lowercase shapes in the `.smcp` glyph spots, negating the value of having an "SC" sibling font. Additionally, the non-SC font includes all the smallcaps, anyway. Based on other fonts with "SC" siblings, neither sibling should contain glyphs marked as smallcaps. More detail here: [clauseggers/Playfair-Display/issues/18](https://github.com/clauseggers/Playfair-Display/issues/18).

## Expected behavior

A FontBakery check could check for common smallcap suffixes in the glyph list, and throw a `WARN` or maybe even a `FAIL` if a font with name substring `" SC"` contains any glyphs with glyphname substring `.sc` or `.smcp` (common suffixes given to smallcap glyphs). Logically, if it's an "SC" font, it should not have any need for smallcap variations.

A possible second check could also check for smallcap suffixes in a font and throw a WARN/FAIL if there is no "SC" sibling font.

**Comment by felipesanches** [comment]
All of Font Bakery's `"new check proposal"` issues are being either closed or migrated to the [Fontspector issue tracker](https://github.com/fonttools/fontspector/issues), as there's no intention to implement new checks on the Font Bakery codebase anymore.

If you believe this proposal is still valid, please post a message here asking for it to be migrated. Otherwise, on **July 20, 2025** (30 days from now) it will be closed due to inactivity.

**Comment by felipesanches** [comment]
The fontspector issue tracker does not currently use priority/importance labels, but when it was hosted at the fontbakery issue tracker:

* @felipesanches applied "P2 Important" on Dec 20, 2018

**Comment by simoncozens** [comment]
I don't think this is actionable. We could (and do) create an SC font by baking in smcp substitutions, which would leave the glyphs with the .sc suffixes.
